### DISCUSSION TITLE
Moonraker problem

### DISCUSSION THREAD

**Discussion by pollosud** [xblax/flashforge_ad5m_klipper_mod#138] [General]
Hello, I installed klipper 0.004. I have some problem with moonraker.
when I digit the ip andress of my printer there is this message.

![Screenshot 2024-04-11 201608](https://github.com/xblax/flashforge_ad5m_klipper_mod/assets/160148315/32e39e92-f6c6-4d2a-be88-ae2d5f4a0679)

**Comment by KaruroChori** [comment]
I see no issue, that is how moonraker is supposed to be.

**Comment by pollosud** [comment]
It works thanks
